model/task/task.go

```diff
@@ -1394,16 +1397,21 @@ func ResetTasks(taskIds []string) error {
 			"$set": bson.M{
 				ActivatedKey:           true,
 				SecretKey:              utility.RandomString(),
+				HostIdKey:              "",
```

Comment:
Can you instead add this to the $unset list?

Reply:
Wouldn't $unset still set it to the empty string though? I'm not sure I understand the difference.

Reply:
it'll remove the field from the document